yt/visualization/plot_window.py

```diff
@@ -1267,6 +1269,41 @@ def show_axes(self, field=None):
             self.plots[f].show_axes()
         return self
 
+    def export_to_mpl_figure(self, nrows_ncols, axes_pad=1.0, cbar_pad="0%"):
```

Comment:
Could you either cover *all* possible arguments from ImageGrid, or bundle them all in a `**kwarg` ?
I'd be in favour of the latter, provided the docstrings explicitly refers to ImageGrid's documentation, but the former would be equally satisfying and arguably clearer.

Reply:
sure.  I was going to add options, but kwarg is perhaps cleanest.  I'm leaving this as at draft as it is something I just hacked up during the yt workshop today and I'll continue hacking on it during the workshop.

Reply:
Ah sorry, I didn't see the "draft flag" (nor did I knew its existence. Neat) !

I don't know whether kwargs is cleaner, but it has the advantage that we won't need to maintain new arguments if mpl ever adds some.

Reply:
I've added the full set of ImageGrid options and opened this for review